### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by clumens**

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+
+Preparing local development environment for Haskell
+===================================================
+
+For development we use the latest upstream versions:
+
+1) Remove the standard `haskell-platform` and `ghc-*` RPMs if you have them installed
+2) Download version **8.0.2** of the generic Haskell Platform distribution from
+   https://www.haskell.org/platform/linux.html#linux-generic
```
Comment (bcl) [inline DEVELOPMENT.md]: The current latest version is 8.4.2 not 8.0.2

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+```
+$ cd src/ && cabal sandbox init && cabal install
+```
+
+Executing unit tests
+====================
+
+    $ cabal sandbox init
+    $ cabal install --dependencies-only --enable-tests
+    $ cabal test
```
Comment (bcl) [inline DEVELOPMENT.md]: Have you confirmed these in a clean install? I keep having problems with hlint and happy when I do this inside docker (I think I fixed it last time by installing the fedora version of happy).

File: README.md [hunk 3]
```diff
@@ -1,80 +1,118 @@
-[![Build Status](https://travis-ci.org/weldr/bdcs.svg?branch=master)](https://travis-ci.org/weldr/bdcs)
-[![Coverage Status](https://coveralls.io/repos/github/weldr/bdcs/badge.svg?branch=master)](https://coveralls.io/github/weldr/bdcs?branch=master)
+The bdcs project is the main backend component of welder.  It is responsible for
```
Comment (bcl) [inline README.md]: Are we welder or weldr or Weldr? I'm currently not 100% sure, but think it should match our github name.

File: DEVELOPMENT.md (hunk 1 above)
Comment (clumens) [reply DEVELOPMENT.md]: Fixed.

File: README.md (hunk 3 above)
Comment (clumens) [reply README.md]: I don't know either, but fixed to match the github name.  It's easy enough to change later.

File: DEVELOPMENT.md (hunk 2 above)
Comment (clumens) [reply DEVELOPMENT.md]: I have not - this file was just a straight copy over from what was in README.md before without modification.  However, I have not had problems with this in the past.  I'd be interested in seeing what you're seeing.

**Pull request merged**